#166859372 setup react project

caleb-42 opened the pull request:
#### What does this PR do?
-  Sets up webpack, babel and airbnb eslint

#### Description of Task to be completed?
- install webpack babel dependencies
- install css and js minimizers
- setup airbnb eslint rule
- define env variables
- setup webpack to build react
  
#### How should this be manually tested?
- N/A

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- PT Story link - https://www.pivotaltracker.com/story/show/166859372

Eazybee commented on .eslintrc.json:
```diff
@@ -0,0 +1,29 @@
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
```
I don't think you need disabling these rules because they are kinda important and enforce best practices.

caleb-42 replied:
done

caleb-42 committed:
chore(webpack): setup react project

- install webpack babel dependencies
- install css and js minimizers
- setup airbnb eslint rule
- define env variables
- setup webpack to build react

[Finishes #166859372]
